hackathon_site/dashboard/frontend/src/slices/hardware/cartSlice.ts

```diff
@@ -26,6 +31,73 @@ export const cartReducerName = "cart";
 export const initialState = cartAdapter.getInitialState(extraState);
 export type CartState = typeof initialState;
 
+// Thunk
+interface RejectValue {
+    status: number;
+    message: string[];
+}
+
+export interface OrderResponse {
+    order_id: number;
+    hardware: [
+        {
+            hardware_id: number;
+            quantity_fulfilled: number;
+        }
+    ];
+    errors:
```

Comment:
okay i'm kinda dumb I think if you do 
`{
          hardware_id: number;
          message: string;
 }[]` it's the same thing as what you have here but in the proper format